python/modules/IcePy/BatchRequestInterceptor.cpp

```diff
@@ -121,7 +121,6 @@ batchRequestEnqueue(BatchRequestObject* self, PyObject* /*args*/)
         return 0;
     }
 
-    Py_INCREF(Py_None);
     return Py_None;
```

Comment:
We can use Py_RETURN_NONE if we need to support Python < 3.12. But I think it would be better to support 3.12 and up.

Reply:
I agree, we can require 3.12 and up for 3.8.